griptape/drivers/image_generation/dalle2_image_generation_driver.py

```diff
@@ -0,0 +1,57 @@
+import json
+import os
+import openai
+from typing import Optional
+
+import requests
+from attr import field, Factory, define
+from griptape.artifacts import ImageArtifact
+from griptape.drivers.image_generation.base_image_generation_driver import (
+    BaseImageGenerationDriver,
+)
+
+
+@define
+class Dalle2ImageGenerationDriver(BaseImageGenerationDriver):
```

Comment:
Let's make it a generic `DalleImageGenerationDriver` and manage version at the class property level.

Reply:
Should we prepend with `OpenAi` to match other driver naming schemas?